### DISCUSSION TITLE
How can I make free transform incremental steps?

### DISCUSSION THREAD

**Discussion by ZalgirisKaunas** [clientIO/joint#2736] [Q&A] [answered]
### Introduction

Depending on the shape, my different shapes have to change in size in certain px steps. How can add that to free transform most efficiently? 

I could probably do graph.on('batch:stop', async (evt) and resize cell with Math.round but that I think might affect performance.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment, marked as answer]
You can use the `resizeGrid` option: https://docs.jointjs.com/api/ui/FreeTransform/#resizegrid

**Reply by ZalgirisKaunas** [reply]
neat
